File: 4th semester/Web Programming/practice exam/sampleASP/sampleASP/Models/Project.cs
namespace sampleASP.Models
{
    public class Project
    {
        private int id;
        private int projectManagerId;
        private String name;
        private String description;
        private String members;

        public Project(int id, int projectManagerId, string name, string description, string members)
        {
            this.id = id;
            this.projectManagerId = projectManagerId;
            this.name = name;
            this.description = description;
            this.members = members;
        }
        int getId()
        {
            return id;
        }

        int getprojectManagerId()
        {
            return projectManagerId;
        }

        String getName()
        {
            return name;
        }

        String getDescription()
        {
            return description;
        }

        String getMembers()
        {
            return members;
        }

        public string toString()
        {
            return getId() + " " + getprojectManagerId() + " " + getName() + " " + getDescription() + " " + getMembers();
        }
    }
}

File: 4th semester/Web Programming/practice exam/sampleJSP/src/main/java/ro/ubb/sampleJSP/Domain/Project.java
package ro.ubb.sampleJSP.Domain;

public class Project {
    private int id;
    private int ProjectManagerID;
    private String name;
    private String description;
    private String members;

    public Project(int id, int projectManagerID, String name, String description, String members) {
        this.id = id;
        ProjectManagerID = projectManagerID;
        this.name = name;
        this.description = description;
        this.members = members;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getProjectManagerID() {
        return ProjectManagerID;
    }

    public void setProjectManagerID(int projectManagerID) {
        ProjectManagerID = projectManagerID;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getMembers() {
        return members;
    }

    public void setMembers(String members) {
        this.members = members;
    }
}
